uptime-service-backend/values.yaml

```diff
@@ -2,33 +2,46 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-# Application specific values
+# -- The number of pods to be deployed for bot
+replicaCount: 1
```

Comment:
**nitpick**: I prefer the naming `replicas` because that's the name of the kubernetes field. We have a mix of `replicaCount` and `replicas` in our charts.

Reply:
that was my initial thought as well, but then I tried doing a `helm create chart-example` and  saw it generated `replicaCount` in `values.yaml` and `replicas: {{ .Values.replicaCount }}` in deployment.yaml 
So I would prefer keeping this as it is